src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWalletActor.scala

```diff
@@ -198,6 +197,9 @@ class ErgoWalletActor(settings: ErgoSettings) extends Actor with ScorexLogging {
 
     case ReadTrackedAddresses =>
       sender() ! trackedAddresses.toIndexedSeq
+
+    case ReadInputsFor(outputs) =>
```

Comment:
I don't like that transaction generator requires new functionality in wallet. It is possible to transfer Ergo, transfer assets and issue assets without `ReadInputsFor` request, correct? Why do you need it to generate transactions?

Reply:
@catena2w, to issue new asset we need to know the first input for transaction in advance - so this functionality is required.